src/reader.ts

```diff
@@ -402,26 +403,48 @@ export default class D2Reader {
   };
 
   private isTTSIncrementable(
-    incremental: UserSettingsIncrementable | TTSIncrementable
+    incremental:
+      | UserSettingsIncrementable
+      | TTSIncrementable
+      | MediaOverlayIncrementable
   ): incremental is TTSIncrementable {
     return (
       incremental === "pitch" ||
       incremental === "rate" ||
       incremental === "volume"
     );
   }
+  private isMOIncrementable(
+    incremental:
+      | UserSettingsIncrementable
+      | TTSIncrementable
+      | MediaOverlayIncrementable
+  ): incremental is MediaOverlayIncrementable {
+    return (
+      incremental === "pitch" ||
+      incremental === "rate" ||
+      incremental === "volume"
+    );
+  }
 
   /**
    * Used to increase anything that can be increased,
    * such as pitch, rate, volume, fontSize
    */
   increase = async (
-    incremental: UserSettingsIncrementable | TTSIncrementable
+    incremental:
+      | UserSettingsIncrementable
+      | TTSIncrementable
+      | MediaOverlayIncrementable
   ) => {
     if (this.isTTSIncrementable(incremental)) {
       if (this.navigator.rights?.enableTTS) {
         await this.ttsSettings.increase(incremental);
       }
+    } else if (this.isMOIncrementable(incremental)) {
+      if (this.navigator.rights?.enableMediaOverlays) {
+        await this.mediaOverlaySettings.increase(incremental);
+      }
     } else {
       await this.settings.increase(incremental);
     }
```

Comment:
If both `MediaOverlaySettings` and `TTSIncrementable` use `"volume"`, how does this function know whether you are trying to increase one or the other? Currently if you call `d2Reader.increase("volume")`, only the `TTS` volume is actually increased. Are both meant to be increased or should we have `"TTS_volume"` and `"MediaOverlay_volume"`?

Reply:
ohh that's a good point, i'll test and see what happens

Reply:
okey, so here is what i'll do, i'll rename the `volume` to `mo_volume` in media overlay, since that hasn't yet been used, but changing the tts names will break current implementations.